### PR TITLE
Setup and Mobile First 

### PR DESCRIPTION
**Pull request by sahlminkok**
### In this project we:
1. Add Linter files
2. Add HTML and CSS for mobile_first

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700;900&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+  <nav class="navigation">
+    <a href="#"><img src="images/cabsal.png" id="logo" alt="my logo"></a>
+    <a href="#"><img src="images/Icon - Menu.png" id="menu" alt="Menu"></a> 
+  </nav>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required that this header should have a container section called `header` tag. The header tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. You can check this [article](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.
The expected structure should be like below.

```
 <header class="navigation">
    <a href="#">
      <img src="images/cabsal.png" id="logo" alt="my logo">
    </a>
    <nav>
      <a href="#">
        <img src="images/Icon - Menu.png" id="menu" alt="Menu"> 
      </a>
    </nav> 
  </header>
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  <section class="Header">
```
Comment (Qoosim) [inline index.html]: - [OPTIONAL] I would recommend you to change the class name to something else for consistency sake. something like `headline-section` could be a better optional.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @sahlminkok, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by sahlminkok** [comment]
@Qoosim I have fixed the issues.

**Review by levy002 — APPROVED** [review]
## Hello @sahlminkok 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Zuhaib042 — COMMENTED** [review]
Hi @sahlminkok 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a pull request for another project. kindly submit the PR for the  HTML + basic JavaScript Capstone project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**